app/views.py

```diff
@@ -6,7 +6,7 @@
 from flask_login import current_user, login_user, logout_user, login_required
 from app import app, db, mail
 from app.models import User, Question, Option, Answer, Response
-from app.forms import LoginForm, RegistrationForm, ContactForm, ResetPasswordForm, NewPasswordForm, CreateQnForm
+from app.forms import LoginForm, RegistrationForm, ContactForm, ResetPasswordForm, NewPasswordForm, CreateQnForm, DeactivateForm
 from app.questions import get_question_options, submit_response
 from app.email import register, resend_conf, send_contact_email, send_reset_email
```

Comment:
`send_deactivate_email` not added here.
```suggestion
from app.email import register, resend_conf, send_contact_email, send_reset_email, send_deactivate_email
```